src/lib/Utils/sessionDB.ts

```diff
@@ -33,7 +33,7 @@ if (typeof port === 'number' && !isNaN(port) && typeof host === 'string') {
 
 	sessionDB.on('connect', () => console.log('SessionDB connected'));
 	sessionDB.on('error', (error) => {
-		console.log(error);
+		console.log('[SessionDB]', error);
```

Comment:
Je weet hoe ik er over denk 🙂 🙂 🙂 🙂 🙂 🙂 🙂 🙂 🙂 🙂 🙂 🙂

Reply:
Zodra alles werkt en door getest is haal ik ze eruit 😘